### PR TITLE
Finally fix the grey security pic bug!

### PR DESCRIPTION
**Pull request by zfox23**
Finally fixes [MS10839](https://highfidelity.manuscript.com/f/cases/10839)! Hopefully this is the final time we see this bug surface.

### CODE REVIEW

**Comment by zfox23** [comment]
## Test Plan
### Prerequisites
- You EITHER:
    1. DO NOT already have a wallet associated with your HiFi account OR
    2. Have renamed the `.hifikey` file associated with your account to `<xxx>.hifikey.backup` (if you choose this option, it's assumed that you know how to set up a new wallet from the Wallet Setup screen)
- You must have an HMD

### Tests
- The goal of this test plan is to attempt (unsuccessfully) to get the Wallet Security Pic to show up as a grey square or not show up at all. You must attempt to do so in both HMD and Desktop modes.
- There is no way to reliably reproduce the "grey security pic" bug. Thus, navigating throughout the Commerce system is the only way to attempt to reproduce the bug.

Spend some time doing the following (as well as anything else that you can think of) in both HMD and Desktop modes in an attempt to reproduce the "grey security pic" bug:
- Creating a new wallet
- Changing your security pic
- Changing your password
- Viewing out the Wallet, Checkout, and My Purchases screens
- Switching between HMD and Desktop modes in the same session
- Switching High Fidelity account logins
- Switching domains

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8645/app_12840_PR.apk

File: interface/src/commerce/Wallet.cpp [hunk 1]
```diff
@@ -13,8 +13,9 @@
 #include "Ledger.h"
 #include "Wallet.h"
 #include "Application.h"
-#include "ui/ImageProvider.h"
+#include "ui/SecurityImageProvider.h"
```
Comment (davidkelly) [inline interface/src/commerce/Wallet.cpp]: 👍

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8645/HighFidelity-Beta-PR12840-eab94909e4fbe37926015415c61b348c61b097d3.exe

**Comment by howard-stearns** [comment]
closed in favor of https://github.com/highfidelity/hifi/pull/12862

**Pull request closed without merging**